pallets/runtime/common/src/runtime.rs

```diff
@@ -1016,6 +1024,36 @@ macro_rules! runtime_apis {
                 ) -> AffirmationCount {
                     Settlement::affirmation_count(instruction_id, portfolios)
                 }
+
+                #[inline]
+                fn get_transfer_report(leg: Leg, skip_locked_check: bool) -> Result<(), Vec<DispatchError>> {
```

Comment:
I don't really like using a `Vec` for the error variant like this.  I think there are some jsonrpc clients that might not handle this kind of error very well.